regcomp.c

```diff
@@ -23700,14 +23700,17 @@ S_parse_uniprop_string(pTHX_
         {
             /* Find the property.  The table includes the equals sign, so we
              * use 'j' as-is */
```

Comment:
I think this comment used to refer to the use of `j` in the `do_uniprop_match()` call. Was it supposed to stay just above that?

Reply:
yes